candle-examples/src/lib.rs

```diff
@@ -14,6 +14,10 @@ pub fn device(cpu: bool) -> Result<Device> {
 
 #[cfg(test)]
 mod tests {
+    use anyhow::Result;
```

Comment:
I'm starting to be a bit concerned about the amount of testing that ends up being done here. Culd we put this somewhere under `candle-book` instead, even if it requires an additional `Cargo.toml` there?

Reply:
I managed to overlap a crate and the actual book.
Both are in the same directory yet are 2 different sets of files.